add_to_app/books/README.md

```diff
@@ -36,10 +38,13 @@ page.
   * If the `schema.dart` is modified, the generated classes can be updated with
 
       ```bash
-      flutter pub run pigeon \
-          --input pigeon/schema.dart \
-          --java_out ../android_books/app/src/main/java/dev/flutter/example/books/Api.java \
-          --java_package "dev.flutter.example.books"
+      flutter pub run pigeon --input pigeon/schema.dart \
```

Comment:
Would be nice to bring this into the config function so it's easy to generate (and harder to make mistakes) for someone who just downloaded this, changed a line and want to try again.